src/test/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerIntegrationTest.java

```diff
@@ -1052,6 +1055,14 @@ private void produceSyntheticMessages(String topic) {
     producer.close();
   }
 
+  //TODO - remove / refactor
+  private Producer<byte[], byte[]> createKafkaProducer() {
```

Comment:
Should we default all tests to SSL?

Reply:
no. this method is rather hacky and creates an ad-hoc producer<byte[], byte[]> where the rest of the harness code creates <string, string>, and so should be cleaned up somehow.
however, our tests currently rely on this heavily nd i didnt want ot make this PR any bogger.